docs/quickstart.md

```diff
@@ -462,20 +462,19 @@ import numpy as np
 import sympy as sp
 from pyccel import lambdify
 
-x = sp.Symbols('x')
+x = sp.Symbol('x')
 expr = x**2 + x*5
 f = lambdify(expr, {x : 'float'})
 print(f(3.0))
 
 expr2 = x-x
-f2 = lambdify(expr, {x : 'float'}, float)
+f2 = lambdify(expr, {x : 'float'}, 'float')
```

Comment:
Do we accept a proper type as well as a string? e.g.
```python
f2 = lambdify(expr, {x : float}, float)
```

Reply:
No. Theoretically it is possible to retrieve the type name with something like `result_type.__name__` but this doesn't give a complete type description for something like `list[float]` (which shouldn't be used for this function but gives us an idea of how `np.ndarray[int, 2]` will work when NumPy decide on the syntax). We could add a list of acceptable types, but I thought the simplest thing was to just avoid this problem for now

Reply:
All right!